README_BUILD.md

```diff
@@ -0,0 +1,66 @@
+# Build instructions for libiio
+
+## Install Prerequisites/Dependancies
+
+Basic system setup
+```shell
+analog@precision:~$ sudo apt-get update
+analog@precision:~$ sudo apt-get install build-essential
+```
+Install Prerequisites
+```shell
+analog@precision:~$ sudo apt-get install libxml2-dev bison flex libcdk5-dev cmake
+```
+Install Backends
+```shell
+analog@precision:~$ sudo apt-get install libaio-dev libusb-1.0-0-dev libserialport-dev libxml2-dev libavahi-client-dev
```

Comment:
libxml2-dev is already in the install prerequisites list. Was it added intentionally to show that it is also a backend dependency?

Reply:
that was a bug/oversight. Thanks for checking, should be OK now.